pkg/config/openshift.go

```diff
@@ -36,21 +36,20 @@ type Cluster struct {
 	Name      string `json:"name"`
 	URL       string `json:"url"`
 	LoginURL  string `json:"loginUrl"`
-	Token     string `json:"token"`
 	Reachable bool   `json:"reachable"`
 	BooberURL string `json:"booberUrl"`
 	GoboURL   string `json:"goboUrl"`
 }
 
 // InitClusters initializes Cluster objects for AOConfig
-func (ao *AOConfig) InitClusters() {
-	ao.Clusters = make(map[string]*Cluster)
+func (aoConfig *AOConfig) InitClusters() {
+	aoConfig.Clusters = make(map[string]*Cluster)
 	ch := make(chan *Cluster)
 	configuredClusters := 0
 
-	for _, cluster := range ao.AvailableClusters {
+	for _, cluster := range aoConfig.AvailableClusters {
 		name := cluster
-		urls, err := ao.GetServiceURLs(name)
```

Comment:
Burde vi utlede server-adresser tidligere? Da slipper vi å forholde oss til url patterns i AOConfig.

Reply:
Jeg er enig, og det er det naturlig å ta i egen Jirasak for å forbedre/forenkle ao config. Legger dette til egen sak.